usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -85,6 +89,23 @@
         "custom_schema": "recipient_hash STRING",
         "column_names": [x for x in list(RECIPIENT_PROFILE_DELTA_COLUMNS) if x != "id"],
     },
+    "int.sam_recipient": {
```

Comment:
Since you removed the other `sam_recipient` from `load_table_to_delta` there shouldn't be a need to have `int.` in the name.

Reply:
naming convention simplified.